### PR TITLE
Fix/css related issue

### PR DESCRIPTION
**Pull request by jennieramida**

### CODE REVIEW

File: src/lib/components/forms/TextInput.tsx [hunk 1]
```diff
@@ -78,13 +78,15 @@ export const TextInput = ({
...
         <FormErrorMessage className="error-text">{error}</FormErrorMessage>
       ) : (
-        <FormHelperText className="helper-text">
-          {status?.message ? (
-            getResponseMsg(status, helperText)
-          ) : (
-            <Text color="text.dark">{helperText}</Text>
-          )}
-        </FormHelperText>
+        helperText && (
```
Comment (songwongtp) [inline src/lib/components/forms/TextInput.tsx:81]: Might have a problem if we only have status but no helpertext

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: src/lib/pages/query/components/QueryArea.tsx [hunk 2]
```diff
@@ -89,17 +89,17 @@ export const QueryArea = ({
 
   return (
     <Flex direction="column">
-      <Box width="full" my="16px" alignItems="center">
+      <Box width="full" mt="4" mb="8" alignItems="center">
         {contractAddress && (
-          <Text variant="body3" mb="8px">
+          <Text variant="body3" mb="2">
```
Comment (poomthiti) [inline src/lib/pages/query/components/QueryArea.tsx:94]: I prefer to write mb={2} to differentiate between arbitrary unit ("8px") and chakra space format (2)

File: src/lib/pages/query/components/QueryArea.tsx [hunk 3]
```diff
@@ -89,17 +89,17 @@ export const QueryArea = ({
...
             Message Suggestions:
           </Text>
         )}
         {cmds.length ? (
           <ButtonGroup
             width="90%"
             flexWrap="wrap"
-            rowGap="8px"
+            rowGap="2"
```
Comment (poomthiti) [inline src/lib/pages/query/components/QueryArea.tsx:102]: ditto

File: src/lib/layout/navbar/index.tsx [hunk 4]
```diff
@@ -30,9 +30,16 @@ const Navbar = observer(({ isExpand, setIsExpand }: NavbarProps) => {
...
+        switch (slug) {
+          case "/":
+            return pathName === `/${network}`;
+          case "/contract-list":
+            return pathName === `/${network}${slug}`;
+          case "/public-project":
+            return pathName === `/${network}${slug}`;
+          default:
+            return pathName.includes(`/${network}${slug}`);
+        }
```
Comment (poomthiti) [inline src/lib/layout/navbar/index.tsx:42]: add comment for each case e.g. Handle path params, Handle query params

File: src/lib/layout/navbar/index.tsx [hunk 5]
```diff
@@ -30,9 +30,16 @@ const Navbar = observer(({ isExpand, setIsExpand }: NavbarProps) => {
...
-        return slug === "/"
-          ? pathName === `/${network}`
-          : pathName === `/${network}${slug}`;
+        switch (slug) {
+          case "/":
+            return pathName === `/${network}`;
+          case "/contract-list":
+            return pathName === `/${network}${slug}`;
+          case "/public-project":
+            return pathName === `/${network}${slug}`;
```
Comment (poomthiti) [inline src/lib/layout/navbar/index.tsx:39]:
```tsx
case "/contract-list":
case "/public-project":
   return pathName === `/${network}${slug}`;
```

**Review by poomthiti — CHANGES_REQUESTED** [review]

**Comment by poomthiti** [comment]
Change handleOnClick in PublicProjectCard to this
```ts
const handleOnClick = () => {
      navigate({ pathname: `/public-project/[slug]`, query: { slug } });
    };
```

File: src/lib/layout/navbar/index.tsx [hunk 6]
```diff
@@ -30,9 +30,16 @@ const Navbar = observer(({ isExpand, setIsExpand }: NavbarProps) => {
   const isCurrentPage = useCallback(
```
Comment (evilpeach) [inline src/lib/layout/navbar/index.tsx:30]:
```tsx
  const isCurrentPage = useCallback(
    (slug: string) => {
      const networkPath = network ? `/${network}` : "";

      switch (slug) {
        // handle home page
        case "/":
          return pathName === `${networkPath}`;
        // handle contract list page and public project page
        case "/contract-list":
        case "/public-project":
          return pathName === `${networkPath}${slug}`;
        // handle page with query param
        default:
          return pathName.includes(`${networkPath}${slug}`);
      }
    },
    [network, pathName]
  );
```

**Review by evilpeach — APPROVED** [review]
lgtm

**Pull request merged**